### ISSUE TITLE
Feature Request - Patch Management Titles

### ISSUE THREAD

**Issue by pwadenz** [jamf/Replicator#110]
It would be great to have the ability to copy Patch Management titles between servers. In particular, we just want to copy Patch Management titles supplied by Jamf as the source. This would save us a lot of time when setting up new tenants - bonus points if it can bring across the "Show on Dashboard" value/

**Comment by BIG-RAT** [comment]
Should be functioning in v8.0.0 of Replicator.

**Comment by pwadenz** [comment]
Yes, I was able to use this last week and they all copied across which was a huge time saver, thanks 🙏.

One thing I noticed, although this might have been on the EA transfer rather than the patch management transfer, is that titles which have a Jamf supplied EA copied across but their associated EA did not. I then ended up with Notifications in Jamf and had to go and accept the EA for each title.

Not sure if this is expected, or could be improved on?

**Comment by BIG-RAT** [comment]
If it weren't for the required acceptance on the EAs it would be much smoother. Perhaps legal would be ok auto enabling it on a second server if it's been accepting on the first.

**Comment by pwadenz** [comment]
One thing I didn’t try was accepting those T&C’s ahead of running the replication so I’ll give that a try next time. Thanks for the updates - all these additions really help us.
